dom-event-delegation/main.js

```diff
@@ -0,0 +1,12 @@
+var task = document.querySelector('.task-list');
+task.addEventListener('click', handleClick);
+
+var list = document.querySelector('.task-list-item');
+
+function handleClick(event) {
+  console.log(event.target);
+  console.log(event.target.tagName);
+  if (event.target && event.target.matches('button')) {
+    list.remove();
```

Comment:
you want to remove the ancestor which matches the `.task-list-item` selector of the button which was clicked. 
you don't just want to remove the first one which was selected in the list variable